tensorflow_addons/losses/tests/quantiles_test.py

```diff
@@ -81,12 +80,14 @@ def test_zero_weighted():
 
 
 def test_invalid_sample_weight():
-    pytest.skip("Failing. See https://github.com/tensorflow/addons/issues/1202")
     pin_obj = quantiles.PinballLoss()
     y_true = tf.constant([1, 9, 2, -5, -2, 6], shape=(2, 3, 1))
     y_pred = tf.constant([4, 8, 12, 8, 1, 3], shape=(2, 3, 1))
     sample_weight = tf.constant([3, 6, 5, 0], shape=(2, 2))
-    with pytest.raises(ValueError, match="weights can not be broadcast to values"):
+    with pytest.raises(
+        (ValueError, tf.errors.InvalidArgumentError),
+        match="(weights can not be broadcast to values|Incompatible shapes)",
```

Comment:
Could you detail why there are two exceptions here and error messages as it's not obvious for the reader?

Reply:
Depending on some combination of python version / tf version it was either raising a ValueError with  "weights can not be broadcast to values" or a InvalidArgumentError with "Incompatible shapes" (python 3.5 I think).

So I check for both unconditionally. I could add an if statement to catch the good exception according to the versions.

Reply:
Great, yeah that's a good idea! It would make it easier for readers to understand the decision if there was a "if" statement somewhere. And possibly to remove it later if we drop support for versions of python/tensorflow. See https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python/21065570 to compare version numbers.  We use LooseVersion in the codebase but it's not the right thing to do because it's undocumented.